plugins/LilyPad/Config.cpp

```diff
@@ -1393,8 +1394,13 @@ int BindCommand(Device *dev, unsigned int uid, unsigned int port, unsigned int s
     if (port > 1 || slot > 3 || padtype >= numPadTypes)
         return -1;
 
-    if (!sensitivity)
-        sensitivity = BASE_SENSITIVITY;
+    if (!sensitivity) {
+        if (((uid >> 16) & 0xFF) == ABSAXIS) {
+            sensitivity = 87183;
```

Comment:
You should probably use a constant to represent this (maybe DEFAULT_ANALOG_SENSITIVITY or any other suitable name).